sw_albedo/create_artifact.jl

```diff
@@ -33,28 +42,36 @@ for (file_path, file_url) in zip(FILE_PATHS, FILE_URLS)
         # The server has poor certificates, so we have to disable verification
         downloader = Downloads.Downloader()
         downloader.easy_hook =
-            (easy, info) -> Downloads.Curl.setopt(
-                easy,
-                Downloads.Curl.CURLOPT_SSL_VERIFYPEER,
-                false,
-            )
+            (easy, info) ->
+                Downloads.Curl.setopt(easy, Downloads.Curl.CURLOPT_SSL_VERIFYPEER, false)
         println(file_url)
         println(file_path)
-        downloaded_file = Downloads.download(file_url; downloader, progress = download_rate_callback())
+        downloaded_file =
+            Downloads.download(file_url; downloader, progress = download_rate_callback())
         Base.mv(downloaded_file, file_path)
     end
 end
 
 include("calculate_sw_alb.jl")
 include("calculate_bareground_alb.jl")
+include("process_ceres_data.jl")
+
+# checks that there are no nans or missing values in the dataset
+function check_ds(path)
+    ds = NCDataset(path)
+    for (_, var) in ds
+        @assert !(NaN in var[:])
```

Comment:
Could we check there is no Inf as well? (in the CERES file it's not always 0 / 0, so sometimes it's Inf.)